samples/javaconfig/saml2login/src/main/java/org/springframework/security/samples/config/SecurityConfig.java

```diff
@@ -40,27 +45,43 @@
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
 	RelyingPartyRegistration getSaml2AuthenticationConfiguration() throws Exception {
-		//remote IDP entity ID
+		// remote IDP entity ID
 		String idpEntityId = "https://simplesaml-for-spring-saml.cfapps.io/saml2/idp/metadata.php";
-		//remote WebSSO Endpoint - Where to Send AuthNRequests to
+		// remote WebSSO Endpoint - Where to Send AuthNRequests to
 		String webSsoEndpoint = "https://simplesaml-for-spring-saml.cfapps.io/saml2/idp/SSOService.php";
-		//local registration ID
+		// local registration ID
 		String registrationId = "simplesamlphp";
-		//local entity ID - autogenerated based on URL
+		// local entity ID - autogenerated based on URL
 		String localEntityIdTemplate = "{baseUrl}/saml2/service-provider-metadata/{registrationId}";
-		//local signing (and decryption key)
+		// local signing (and decryption key)
 		Saml2X509Credential signingCredential = getSigningCredential();
-		//IDP certificate for verification of incoming messages
+		// IDP certificate for verification of incoming messages
 		Saml2X509Credential idpVerificationCertificate = getVerificationCertificate();
-		String acsUrlTemplate = "{baseUrl}" + Saml2WebSsoAuthenticationFilter.DEFAULT_FILTER_PROCESSES_URI;
+		String acsUrlTemplate = "{baseUrl}"
+				+ Saml2WebSsoAuthenticationFilter.DEFAULT_FILTER_PROCESSES_URI;
 		return RelyingPartyRegistration.withRegistrationId(registrationId)
-				.remoteIdpEntityId(idpEntityId)
-				.idpWebSsoUrl(webSsoEndpoint)
+				.remoteIdpEntityId(idpEntityId).idpWebSsoUrl(webSsoEndpoint)
 				.credentials(c -> c.add(signingCredential))
 				.credentials(c -> c.add(idpVerificationCertificate))
 				.localEntityIdTemplate(localEntityIdTemplate)
-				.assertionConsumerServiceUrlTemplate(acsUrlTemplate)
-				.build();
+				.assertionConsumerServiceUrlTemplate(acsUrlTemplate).build();
+	}
+
+	private UserDetailsService userDetailsServiceMockup() {
```

Comment:
We'd like to keep the samples to bare minimum.
I will work with you to find a better place for this configuration and how to test it (I'm working on that right now)

Reply:
Sure, it just felt right for me, to add a sample.
Feel free to create a second example, that just uses the UserDetailService.
Let me know, if I can help!